src/webgl/program.js

```diff
@@ -11,6 +11,8 @@ import {VertexShader, FragmentShader} from './shader';
 import {log, uid} from '../utils';
 import assert from 'assert';
 
+const LOG_PROGRAM_PERF_PRIORITY = 3;
```

Comment:
I think we could increase this priority, even to 1. It shouldn't happen very often and useful to turn on without having all the render logging at >=2.